tasks/gitlab_setup.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Clone gitlab to {{ gitlab_install_path }}
+  git:
+    depth: 1
+    repo: https://gitlab.com/gitlab-org/gitlab-{{ gitlab_version }}.git
+    dest: "{{ gitlab_install_path | mandatory }}"
+    version: "{{ gitlab_version_number | mandatory }}"
+    force: yes
+
+- import_tasks: gitlab_permissions.yml
+
+- import_tasks: gitlab_config_files.yml
+
+- name: Configure Git global settings for git user
+  git_config:
+    name: core.autocrlfs
+    scope: global
+    value: input
+
+- name: Disable 'git gc --auto' because GitLab already runs 'git gc' when needed
+  git_config:
+    name: gc.auto
+    scope: global
```

Comment:
This changes the value of the root user. I doubt that GitLab runs under the root user.

Reply:
fixed